drivers/entropy/entropy_nrf5.c

```diff
@@ -19,8 +72,8 @@ struct rand {
 
 #define RAND_DEFINE(name, len) u8_t name[sizeof(struct rand) + len] __aligned(4)
 
-#define RAND_THREAD_LEN (CONFIG_ENTROPY_NRF5_THR_BUF_LEN + 1)
-#define RAND_ISR_LEN    (CONFIG_ENTROPY_NRF5_ISR_BUF_LEN + 1)
+#define RAND_THREAD_LEN (CONFIG_ENTROPY_NRF5_THR_THRESHOLD + 1)
```

Comment:
This defines the total length of the buffer, which is BUF_LEN. The THRESHOLD is only there to trigger the hardware.

Reply:
Oops, I am really sorry, this patch was not supposed to be included.